trajoptlib/src/RustFFI.hpp

```diff
@@ -21,7 +21,7 @@ static void trycatch(Try&& func, Fail&& fail) noexcept try {
   fail(e.what());
 } catch (const sleipnir::SolverExitCondition& e) {
   // TODO: Use std::to_underlying() from C++23
-  fail(std::to_string(static_cast<uint8_t>(e)));
+  fail(std::to_string(static_cast<int8_t>(e)));
```

Comment:
Also add `#include <type_traits>`.
```suggestion
  fail(std::to_string(static_cast<std::underlying_type_t<decltype(e)>>(e)));
```

Reply:
Can we do that now? I thought we didn't support c++23 fully yet

Reply:
Reread the type more carefully.